crates/collab/src/llm/db/queries/usages.rs

```diff
@@ -376,6 +379,7 @@ impl LlmDatabase {
                 .filter(
                     usage::Column::ModelId
                         .eq(model.id)
+                        .and(usage::Column::MeasureId.eq(tokens_per_minute))
```

Comment:
Should this be `tokens_per_day`?

Reply:
I don't think so, as I think that would make the active user counts snapshotted for the day.

A user wouldn't roll out of the active users count until they hadn't make any requests for 24 hours.